.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "python2-pybind11/pybind11"]
+	path = python2-pybind11/pybind11
+	url = https://github.com/pybind/pybind11.git
```

Comment:
I think I'd rather not do this with git submodules, even if that means the user is responsible for making pybind11 available on their own.  Could you remove them and force-push them away?